nncf/quantization/algorithms/accuracy_control/algorithm.py

```diff
@@ -463,14 +463,14 @@ def _collect_original_biases_and_weights(
         for node in initial_model_graph.get_all_nodes():
             if algo_backend.is_node_with_bias(node, initial_model_graph):
                 node_with_bias = quantized_model_graph.get_node_by_name(node.node_name)
-                node_with_bias.data["original_bias"] = algo_backend.get_bias_value(
+                node_with_bias.attributes["original_bias"] = algo_backend.get_bias_value(
                     node, initial_model_graph, initial_model
                 )
             if algo_backend.is_node_with_weight(node):
                 node_with_weight = quantized_model_graph.get_node_by_name(node.node_name)
                 for port_id in algo_backend.get_weight_tensor_port_ids(node_with_weight):
                     weight = algo_backend.get_weight_value(node, initial_model, port_id)
-                    node_with_weight.data[f"original_weight.{port_id}"] = weight
+                    node_with_weight.attributes[f"original_weight.{port_id}"] = weight
```

Comment:
test?

Reply:
I could add end-to-end for the full algorithm. But anyway it should not stop merge of this PR because current develop branch is not stable.

Reply:
Please add the unit test for this function in the follow up PR.

Reply:
Sure